bundles/org.openhab.binding.modbus.sunspec/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.openhab.addons.bundles</groupId>
+        <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+        <version>2.5.2-SNAPSHOT</version>
```

Comment:
```suggestion
        <version>2.5.3-SNAPSHOT</version>
```

Reply:
hmm, is this correct? Maven can not resolve this version at my environment yet. Also I can not find any binding using version 2.5.3-SNAPSHOT at the moment.

Reply:
Thank you, updated to the new version.